docs/user-guide/monitoring.md

```diff
@@ -0,0 +1,161 @@
+# Prometheus and Grafana installation
+
+This tutorial will show you how to install [Prometheus](https://prometheus.io/) and [Grafana](https://grafana.com/) for the metric scraping from the NGINX Ingress controller.
+
+## Before You Begin
+
+Now, deploy the Ingress controller following the instructions [here](../deploy/index.md).
+
+Note that the yaml files that are used in this tutorial, stored in [deploy/monitoring](https://github.com/kubernetes/ingress-nginx/tree/master/deploy/monitoring) folder in GitHub repository [kubernetes/ingress-nginx](https://github.com/kubernetes/ingress-nginx).
+
+## Deploy and configure Prometheus Server
+
+The Prometheus server is needed to configure so that it can discover endpoints of services. If a Prometheus server is already running in cluster and if it is configured in a way that it can discover service endpoints, no extra configuration will be needed.
+If there is no existing Prometheus server running, rest of this tutorial will create a Prometheus server with appropriate configuration.
```

Comment:
"*the* rest of this tutorial will guide you through the step needed to deploy a properly configured Prometheus server."